index.md

```diff
@@ -425,8 +426,8 @@ Taulukossa on kuvattu sanoman tietueiden käyttö.
 |Nimi|Tyyppi|Käytössä|Kuvaus|
 |:---|:---|:---|:---|
 |InformationRequestOpeningV01| | | |
-|&nbsp;&nbsp;&nbsp;&nbsp;InvstgtnId|Max35Text|Kyllä|Tutkinnan case id|
-|&nbsp;&nbsp;&nbsp;&nbsp;LglMndtBsis|LegalMandate1|Kyllä|Lainsäädäntöperuste. Numeroarvo välillä 100..n. Ensimmäiset numerot kertovat viranomaisen ja kaksi viimeistä numeroa lainsäädäntöperusteen. Lainsäädäntöperusteita kootaan Finanssiala Ry:n kanssa sovitun mukaisesti viranomaisten yhteisesti ylläpitämään kooditaulukkoon. Taulukko toimitetaan tiedonhakujärjestelmän toteuttajille sopimusneuvottelujen yhteydessä (taulukko ei ole julkinen).|
+|&nbsp;&nbsp;&nbsp;&nbsp;InvstgtnId|Max35Text|Ei|Aina "Customs_aggr"|
```

Comment:
Pitäisikö tässä olla kuitenkin "Kyllä" tuossa käytössä sarakkeessa? Vertaa alempi YesNoIndicator.

Reply:
No, the content of these 2 fields are not forwarded to consumer and will always have the value "Customs_aggr", hence not used.

Reply:
I got what you meant now, yes it should be - updated